IngresseSDK/Services/SearchService.swift

```diff
@@ -51,19 +51,20 @@ public class SearchService: BaseService {
     ///   - eventTitle: title from required event search
     ///   - onSuccess: success callback with NewEvent array
     ///   - onError: fail callback with APIError
-    public func getEvents(filters: [String: String],
+    public func getEvents(request: Request.Event.Search,
                           onSuccess: @escaping (_ event: [IngresseSDK.NewEvent], _ totalResults: Int) -> Void,
                           onError: @escaping (_ error: IngresseSDK.APIError) -> Void) {
 
         var builder = URLBuilder(client: client)
```

Comment:
O builder poderia ser imutável agora.
```suggestion
        let builder = URLBuilder(client: client)
```